ASWF/gsoc_2021_README.md

```diff
@@ -0,0 +1,73 @@
+
+## About the Academt Software Foundation
+
+The Academy Software Foundation (ASWF) is collaboration between the Linux
+Foundation and the Academy of Motion Picture Arts and Sciences, serving as
+an umbrella organization that is the home to several key open source
+software projects used by the film industry (primarily visual effects and
+animation production). The projects we manage are some of the most
+high-profile and widely used open source projects originating and used in
+film production studios. It's a great opportunity for students to get
+involved with projects that will be used professionally on films!
+
+The mission of the ASWF is to increase the quality and quantity of
+contributions to the content creation industry's open source software base;
+to provide a neutral forum to coordinate cross-project efforts; to provide a
+common build and test infrastructure; and to provide individuals and
+organizations a clear path to participation in advancing our open source
+ecosystem.
+
+Member projects are focused primarily on visual effects and animation
+production, and at present include 5 distinct software packages. Student
+summer project applications can be for any of these packages individually,
+as well as potentially including tasks that bridge between them or provide
+common infrastructure (they share CI and other resources). The current
+member projects are:
+
+
+## Application Instructions for Students
+
+Please review the CONTRIBUTING guidelines thoroughly before applying. OCIO maintains a contributor license agreement (CLA) which must be signed to accept contributions.
+
+**Successfully submitting a Pull Request to the OCIO project is a strict requirement prior to submitting an application.**
+This is not negotiable. We will not even read applications from students who have not submitted an acceptable PR to the project. It can be a truly minimal change -- for example, fixing a typo in an error message. You will find issues tagged "good first issue" on our GitHub repo, which are ideal candidates for your small trial PR to qualify for application.
+
+## Application template
+
+Please use this application template. Feel free to expand or tell us any other relevant details, but this is a good place to start. The applications will be judged by representatives of the OCIO technical advisory council.
+
+* Student name:
+* School / year / area of study:
+* Relevant courses, materials studied, jobs, programming projects, skills, other open source projects you've contributed to. Go ahead and tell us anything about your background that you think we will find interesting for our projects.
+* Project proposal - please explain in detail and give specifics on acceptance criteria -- how we can measure your success, how we can test your code. Provide a weekly timeline for expected progress.
+* Please tell us which fix you contributed:
```

Comment:
"which fix you contributed:" --> "which test PR you had merged into to the OCIO repo:"